aiosmtpd/tests/test_main.py

```diff
@@ -92,43 +92,59 @@ def setuid(mocker):
 # region ##### Helper Funcs ###########################################################
 
 
-def watch_for_tls(ready_flag, has_tls, req_tls):
-    has_tls.value = False
-    req_tls.value = False
+def watch_for_tls(ready_flag, retq: MP.Queue):
+    has_tls = False
+    req_tls = False
     ready_flag.set()
     start = time.monotonic()
-    while (time.monotonic() - start) <= AUTOSTOP_DELAY:
+    delay = AUTOSTOP_DELAY * 1.5
+    while (time.monotonic() - start) <= delay:
```

Comment:
Make sure that the child process still exists when the `main()` finishes initializing.